content/docs/user-guide/networking/settings.md

```diff
@@ -0,0 +1,139 @@
+| sv_isDedicated          | Whether the host command creates an independent or client hosted server              | True    |       |
+| sv_isTransient          | Whether a dedicated server shuts down if all existing connections disconnect         | True    ||
+ | sv_serverSendRateMs     | Minimum number of milliseconds between each network update                           | 50 ms   || 
+
+### Replication Window Settings
+| Setting                            | Description                                                                               | Default | Notes |
+|------------------------------------|-------------------------------------------------------------------------------------------|---------|-------|
+| sv_ReplicateServerProxies          | Enable sending of ServerProxy entities to clients                                         | true    ||
+| sv_MaxEntitiesToTrackReplication   | The default max number of entities to track for replication                               | 512     ||
+| sv_MinEntitiesToReplicate          | The default min number of entities to replicate to a client connection                    | 128     ||
+| sv_MaxEntitiesToReplicate          | "The default max number of entities to replicate to a client connection"                  | 256     ||
+| sv_PacketsToIntegrateQos           | The number of packets to accumulate before updating connection quality of service metrics | 1000    |       |
+| sv_BadConnectionThreshold          | The loss percentage beyond which we consider our network bad                              | 0.25    ||
+| sv_ClientReplicationWindowUpdateMs | Rate for replication window updates                                                       | 300 ms  ||               
+| sv_ClientAwarenessRadius           | The maximum distance entities can be from the client and still be relevant.               | 500.0   ||
+
+### Rewind Settings
+| Setting                        | Description                                                                     | Default | Notes |
+|--------------------------------|---------------------------------------------------------------------------------|---------|-------|
+| sv_RewindVolumeExtrudeDistance | The amount to increase rewind volume checks to account for fast moving entities | 50      |       |
+
+### Editor Server Settings
+Settings to control how a server should be launched during the "play-in-editor" mode.
+
+| Setting               | Description                              | Default   | Notes |
+|-----------------------|------------------------------------------|-----------|-------|
+| editorsv_port         |                                          | 33450     |       | 
+| editorsv_serveraddr   | The address of the server to connect to. | LocalHost |       |
+| editorsv_process      |                                          | ""        |       |
+| editorsv_rhi_override |                                          | ""        | 
```

Comment:
If editorsv_launch is true the editor will launch its own server. By default it’ll launch the server using the same render hardware interface (rhi). For example, if the editor is using DX12, then the new server will be launched using DX12. Likewise, if the Editor is running tests in headless mode with rhi=null, the server launcher will also be headless. Editorsv_rhi_override can be used to override the rhi. It is common for developers who don’t need to see the Editor’s server graphics to use editorsv_rhi_override=null.

Reply:
Added